### PR TITLE
Fix more eventbus tests

### PR DESCRIPTION
**Pull request by mbaechler**

### CODE REVIEW

**Comment by mbaechler** [comment]
```
[4d9405bf4a269a7b28ce80afa0b47e3db316e8fe] [ERROR] Errors: 
[4d9405bf4a269a7b28ce80afa0b47e3db316e8fe] [ERROR]   LinshareBlobExportMechanismProvidingTest.<clinit>:29 » ExceptionInInitializer
```
usual suspects ... test this please

**Comment by trantienduchn** [comment]
```
[4d9405bf4a269a7b28ce80afa0b47e3db316e8fe] [ERROR] Errors: 
[4d9405bf4a269a7b28ce80afa0b47e3db316e8fe] [ERROR]   RabbitMQEventBusTest$LifeCycleTest$SingleEventBus$DispatchingWhenNetWorkIssue.dispatchShouldWorkAfterNetworkIssuesForOldRegistration:367 » ConditionTimeout
```
test this please

**Comment by mbaechler** [comment]
```
[4d9405bf4a269a7b28ce80afa0b47e3db316e8fe] [ERROR] Errors: 
[4d9405bf4a269a7b28ce80afa0b47e3db316e8fe] [ERROR]   VacationIntegrationTest.jmapVacationShouldNotSendNotificationTwice:240 » ConditionTimeout
```
test this please

**Comment by mbaechler** [comment]
green. Will renamed commit to add JAMES-2659 but we can merge it.

File: mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java [hunk 1]
```diff
@@ -579,8 +580,7 @@ void registrationsShouldNotHandleEventsAfterStop() throws Exception {
 
                     TimeUnit.SECONDS.sleep(1);
                     assertThat(listener.numberOfEventCalls())
-                        .isEqualTo(callsAfterStop)
-                        .isLessThanOrEqualTo(MAX_EVENT_DISPATCHED_COUNT);
+                        .isCloseTo(callsAfterStop, Percentage.withPercentage(2));
```
Comment (rouazana) [inline mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java]: I don't get how it matches the test name: isCloseTo means +/- 2% while the test name tells that it should be inferior to some value.

**Comment by chibenwa** [comment]
```
[5084bafd9d5a1f398c0fc553e46b9d5686e09667] [ERROR]   GetMessageListMethodTest.setup:114 » ElasticsearchStatus Elasticsearch excepti...
```

test this please

**Comment by mbaechler** [comment]
green but deployment tests. mergeable

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**